src/js/common/components/Modal.jsx

```diff
@@ -26,9 +26,11 @@ class Modal extends React.Component {
   componentWillReceiveProps(newProps) {
     if (newProps.visible && !this.props.visible) {
       this.setState({ lastFocus: document.activeElement, focusListener: focusListener(newProps.focusSelector) });
+      document.body.classList.add('modal-open');
```

Comment:
Good call—`componentDidMount` would be a good place for adding the `modal-open` class. Removing the class could still work in this lifecycle method